docs/application/web/guides/applications/addon.md

```diff
@@ -0,0 +1,200 @@
+
+- [Mechanism](#mechanism)
+
+  You can understand how each addon works.
+
+- [Comparing to Chrome Extensions](#compare)
+
+  Addons look similar for Chrome Extensions. However, there are certain differences that you can understand in this section.
+
+- [API](#api)
+
+  To write addons, you can use the predefined set of APIs.  
+
+- [Packaging](#packaging)
+
+  To generate a package for an addon, you can use Tizen Studio. There is a separate tag for addon in config.xml.
+
+- [Management](#management)
+
+  To manage installed addons, you can use a separate Web application named Web App Addon Setting.
+
+<a name="configure"></a>
+## Configuration
+
+### Manifest file (mandatory)
+
+The manifest file includes the metadata for an addon:
+
+  - **name**: Name of the addon.
+  - **version**: Version of the addon.
+  -	**main**: The file name of the main JavaScript.
+  -	**settings**: Not available.
+  -	**description**: Description of the addon.
+
+Following is an example of a manifest file. The filename must be `manifest.json`:
+ 
+    ```
+    {
+        “name”: “addonSampleAd”,
+        "version”: “0.0.1”,
+        “main”: “addonSampleAd”,
+        “settings”: “”,
+        “description”: “Sample to show how to show up advertisement at prelaunch event”
+    }
+    ```
+
+### Main JavaScript File
+
+You must have a main JavaScript file.
+
+The main JavaScript file must have the same name with `main` in the manifest file. The main JavaScript file includes:
+
+  - `activate()`: To register a callback to an addon event.
+  - `deactivate()`: To unregister the callback to the addon event.
+  - `Callback Functions`: You can write your own code to define the behavior that happens when an addon event occurs.
+
+Following is an example of a main JavaScript file:
+
+    ```
+    class AddonSampleAd {
+        constructor(em) {
+          console.log('AddonSampleAd constructor');
+          this.evt = em;
+        }
+        activate() {
+            console.log('AddonSampleAd.activate');
+            this.evt.on('lcPrelaunch', this.prelaunch);
+        }
+        deactivate() {
+            console.log('AddonSampleAd.dectivate');
+            this.evt.off('lcPrelaunch', this.prelaunch);
+        }
+        prelaunch(winId, orgUrl) {
+            console.log('AddonSampleAd.prelaunch');
+        }
+    }
+    module.exports = AddonSampleAd;
+
+    ```
+
+<a name="event"></a>
+## Event
+
+| Event                                      | Description                              |
+| ------------------------------------------ | ---------------------------------------- |
+| lcPrelaunch | Emitted once for Web app launching just before the main page is shown. |
+| lcResume | Emitted when the app is resumed. |
+| lcSuspend | Emitted when the app is suspended. |
+| lcQuit | Emitted before the app is ended. |
+| contentDidFinishLoad | Emitted when the page navigation is done. | 
```

Comment:
For the event names : Don't we need to use \` style here?

For the Description : How about "triggered" or other word instead of "Emitted?"

Reply:
Done for event names.

For the description,
I prefer to use 'emitted' because it uses the method 'Emit()' of object 'EventEmitter' in the implementation to trigger events. What do you think about this ?

Reply:
- event names: done

- description: I want to keep as it is because, in its implementation, it uses Emit() of EventEmitter.  Therefore, I think 'Emitted' seems to be more proper.
    Please let me know your opinion.

Reply:
Thanks for the answer. If so, I also agree that 'Emitted' is proper.